sig-apps/annual-report-2021.md

```diff
@@ -4,134 +4,120 @@
+     - [2214 - Indexed Job](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/2214-indexed-job/README.md) - 1.22
+     - [2232 - Suspend Job](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/2232-suspend-jobs/README.md) - 1.22
+     - [2255 - ReplicaSet Pod Deletion Cost](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2255-pod-cost/README.md) - 1.22
+     - [2307 - Job tracking without lingering Pods](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/2926-job-mutable-scheduling-directives/README.md) - 1.23
+     - [2599 - minReadySeconds for StatefulSets](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md) - 1.23
+     - [2926 - Mutable Node Scheduling Directives for Jobs](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md) - 1.23
    - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-   - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+     - [2185 - Random Pod Selection on ReplicaSet Downscale](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/2185-random-pod-select-on-replicaset-downscale/README.md) - 1.21
+     - [1591 - Allow DaemonSets to surge during update like Deployments](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/1591-daemonset-surge/README.md) - 1.21
+     - [2214 - Indexed Job](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/2214-indexed-job/README.md) - 1.21
+     - [2232 - Suspend Job](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/2232-suspend-jobs/README.md) - 1.21
+     - [2255 - ReplicaSet Pod Deletion Cost](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2255-pod-cost/README.md) - 1.21
+     - [2307 - Job tracking without lingering Pods](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2307-job-tracking-without-lingering-pods/README.md) - 1.22
+     - [2599 - minReadySeconds for StatefulSets](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md) - 1.22
+     - [1847 - Auto delete PVCs created by StatefulSet](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/1847-autoremove-statefulset-pvcs/README.md) - 1.23
+     - [2879 - Track ready Pods in Job status](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/2879-ready-pods-job-status/README.md) - 1.23
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - Growing additional reviewers and approvers, also see no. 4 below.
```

Comment:
Has it been considered to have approvers per controller or set of controllers?

For example, I would be interested in becoming an approver for job and cronjob. But since today I also have to be an approver for the rest of the controllers (which I'm not fully familiar with), that sounds like a big time investment.

Reply:
It is an option, we don't expect anyone to have familiarity with all the controllers, and I do keep you at the top of the list for approver promotion in the near future :wink: